cluster/gce/gci/configure-helper.sh

```diff
@@ -3019,30 +3019,37 @@ EOF
     fi
   fi
   cat > "${config_path}" <<EOF
+version = 2
+# Kubernetes requires the cri plugin.
+required_plugins = ["io.containerd.grpc.v1.cri"]
```

Comment:
we don't need this line anymore i think (`required_plugins`)

Reply:
there are more differences with the one that lives in containerd/containerd
let's get this fixed and follow up on another PR

Reply:
we can deal with this in another PR